docs/test-plan.md

```diff
@@ -189,9 +192,28 @@ Are there any components that are not showing up?
 
 ## Optional Test Cases
 
-These are mostly aimed at MRVA, but some of them are also applicable to non-MRVA queries.
-
-### Selecting repositories to run on
+### Modeling Flow
+
+2. Check that a method can have multiple models:
```

Comment:
The linter has spotted this needs to start with `1`, though I believe it won't actually affect the rendering it'll be nice to be consistent.